src/components/Resources/Quiz/Categories/QuizPage.jsx

```diff
@@ -150,30 +168,79 @@ export default function QuizPage() {
                             </QuestionCount>
                             <QuestionText>{questions[currentQuestion].questionText}</QuestionText>
                             {showAnswer && disableSkipButton ? (
-                                <SkipButton onClick={() => handleSkipButton(questions.length)}>
-                                    nextQuestion {`>>>`}
-                                </SkipButton>
+                                <div
+                                    style={{
+                                        display: "flex",
+                                        flexDirection: "row",
+                                        alignSelf: "end",
+                                        margin: "10px",
+                                        maxWidth: "100%",
+                                    }}
+                                >
+                                    {ShowAdditionalInfo && showAnswer ? (
+                                        <PopUpWindow onClose={handleClosePopup}>
+                                            <div
+                                                style={{
+                                                    backgroundColor: "#282828",
+                                                    padding: "30px",
+                                                    borderRadius: "5px",
+                                                }}
+                                            >
+                                                afezfbfibzfbzbfezbfbezzfbfbijbbfzfff iueb f f hfen c v v e vbe vebvv
+                                                nnffbfefbjfnefefnejnfjenff fn fnuf i jafinevjnzkfdiezjfnz vj ejf vef ee
+                                                vfeunejiuir rencnerc freuer vuire reinjznjzeof zef efiiv rjr f ervne f
+                                                nvevf vfenv rf revevne fn reinvev evue vebve jveiuv h
+                                                jnrjfnerjnjnijnegene vnv e vri jner v jnf erfref v feij ionz ezf &jv vi
+                                                invenvvjev vnvevev iefenrbvnjre e r en n iuznfjnerzfv nn n v{" "}
+                                                {questions[currentQuestion].additionalInfo}
```

Comment:
```suggestion
                                                {questions[currentQuestion].additionalInfo}
```
for now it is enough like this :)